BitwardenShared/UI/Vault/Vault/VaultList/VaultListProcessor.swift

```diff
@@ -86,7 +86,7 @@ final class VaultListProcessor: StateProcessor<
         case .refreshAccountProfiles:
             await refreshProfileState()
         case .refreshVault:
-            await refreshVault(isManualRefresh: true)
+            await refreshVault(isManualRefresh: false)
```

Comment:
🤔 `isManualRefresh` acts alongside with whether the user has `allowSyncOnRefresh` enabled or not. 
IMO if the user has `allowSyncOnRefresh` turned on then they would expect the vault to be synced (forced) when manually refreshing. Also, as I understand it that's what the [Bitwarden Help center says](https://bitwarden.com/help/vault-sync/#manual-sync):
> To manually sync your vault from a Bitwarden app:
Open the  Settings tab, tap the Other option, and tap Sync now.
Toggle the Enable sync on refresh option to allow your vault to be synced using a pulldown gesture on the  My Vault tab.

If we do this and let manually refreshing to have the timing logic then, e.g., if the user is part of an organization where some items were updated some minutes after the last sync and the user manually refreshes the view they won't see the updated items and they will think something is not working.
Furthermore, this would have a different behavior here than in `VaultGroup` and `Send`.

Reply:
Oh roger, now I've read the conversation it makes sense; thanks for the context!